Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workbench/PublicPage.java

```diff
@@ -0,0 +1,121 @@
+package edu.asu.spring.quadriga.domain.impl.workbench;
+
+import java.util.Date;
+import java.util.List;
+
+/**
+ * @description : Project class describing the properties of a Public Page object
+ * 
+ * @author : Prasanth Priya Nesan
+ * 
+ */
+
+import org.springframework.stereotype.Service;
+
+import edu.asu.spring.quadriga.domain.workbench.IPublicPage;
+
+public class PublicPage implements IPublicPage {

```

Comment:
Let's create a separate class for the title/description pairs. Otherwise we have to change this class if we want to be able to add more than 3 text blocks on a website. Or you could just make this class "PublicPageTextBlock" with one title and description field.